tests/kafkatest/services/verifiable_producer.py

```diff
@@ -187,9 +197,21 @@ def _has_output(self, node):
         except RemoteCommandError:
             return True
 
+    def get_user_consumer_properties(self):
+        consumer_properties = ""
+
+        for key,value in self.consumer_properties.iteritems():
+            consumer_properties += " --consumer-property %s=%s" % (key, value)
```

Comment:
Same thing as above.

Reply:
I verified it work with kafka-console-consumer too.